app/components/containerInspect/containerInspectController.js

```diff
@@ -1,12 +1,12 @@
-angular.module('containerInspect', [])
+angular.module('containerInspect', ['angular-json-tree'])
 .controller('ContainerInspectController', ['$scope', '$transition$', 'Notifications', 'ContainerService',
 function ($scope, $transition$, Notifications, ContainerService) {
   function initView() {
     $('#loadingViewSpinner').show();
-    
+
     ContainerService.inspect($transition$.params().id)
     .then(function success(d) {
-      $scope.containerInfo = d;
+      $scope.containerInfo.object = d;
```

Comment:
What's the point of creating an inner object called `object` inside `$scope.containerInfo` ? I'd rather keep it as it was before: `$scope.containerInfo = d;`

Reply:
There is no point at all. But, I don't know why, angular-json-tree won't work if `$scope.containerInfo = d;` is used. Instead of expanding the tree, all the JSON structure is shown in a single field:

![untitled](https://user-images.githubusercontent.com/6628437/31908429-21e33b1e-b837-11e7-9564-213bfd837305.png)

Reply:
See my comment above.